documentation/index.md

```diff
@@ -571,7 +574,7 @@ def pipe(x: bool, y: bool):
 assert pipe(True, False) == (False, True)
 ```
 ### **Conditional Execution**
-Currently, conditional statements are not supported in `DAG`. However, `ExecNode`s can be executed conditionally by passing a `bool` to the added parameter `twz_active`. This parameter can be a constant or a result of an execution of other `ExecNode` in the `DAG`. Since basic boolean operations are implemented on `UsageExecNode`s, you can use bitwise operations (`&`, `or`) to simulate `and`, `or`; however this is not recommended, please use the provided `and_`, `or_` and `not_` `ExecNode`s instead.
+`ExecNode`s can be executed conditionally by passing a `bool` to the added parameter `twz_active`. This parameter can be a constant or a result of an execution of other `ExecNode` in the `DAG`. Since basic boolean operations are implemented on `UsageExecNode`s, you can use bitwise operations (`&`, `or`) to simulate `and`, `or`; however this is not recommended, please use the provided `and_`, `or_` and `not_` `ExecNode`s instead.
```

Comment:
I'm actually considering renaming the and_ or_ not_ functions to xn_and, xn_or, xn_not as imo it's clearer

Reply:
let's do it in another PR